### PR TITLE
Hello Microverse Project Pull Request

### PR DESCRIPTION
**Pull request by gitbygaurav**
This is a simple html-css based project here we have used h1 tag and given it a color of yellow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+- HTML
+- CSS
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
```
Comment (mckent05) [inline README.md]: - In order to make your readme file look more professional, I'd suggest that yu include steps on how your project can be cloned locally, please try to fix this 👍 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @gauravgangwarweb,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
almost there

## Highlights
- You have set up your project well 👍 👍 
- All linter check are passing correctly ✔️ 
- Your repo looks professional 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (Debas-31) [inline README.md:45]: [OPTIONAL] Kindly check the link for the project MIT is not working. Please fix it :wrench: so the readme file is simple, readable, and professional. :smiley_cat:

**Review by Debas-31 — APPROVED** [review]
>Hi @gauravgangwarweb :wave: 

 # Status: Approved :green_circle: 

Great work on making the changes requested by a previous reviewer. Your project is complete !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- GitHub flow :heavy_check_mark:
- Project working::heavy_check_mark:
- Good commit message :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**